### PR TITLE
Panel upgrades

### PR DESCRIPTION
**Pull request by msecret**
Changes typography in the panel to match new designs, adding colors,
etc.

Should be minor version change.

### CODE REVIEW

File: src/css/components/all.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 @import 'actions.scss';
 @import 'activity_log.scss';
 @import 'billboard.scss';
+@import 'confirm';
```
Comment (jcscottiii) [inline src/css/components/all.scss]: `confirm.scss`

File: src/css/components/confirm.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.confirm {
+  width: 100%;
+}
+
+.confirm-nexto {
```
Comment (jcscottiii) [inline src/css/components/confirm.scss:10]: `confirm-next-to`?

File: src/css/components/actions.scss [hunk 3]
```diff
@@ -17,23 +17,54 @@
...
+      box-shadow: inset 0 0 0 2px $color-warning;
+    }
+  }
+
+  &.action-finish {
+    background-color: $color-finish;
+  }
 }
 
+/* TODO refactor all actions */
```
Comment (jcscottiii) [inline src/css/components/actions.scss]: refactor how so? or put a link to an issue here.
thinking about if another dev came in, would they know what to do

File: src/css/components/confirm.scss (hunk 2 above)
Comment (msecret) [reply src/css/components/confirm.scss:10]: It should be that because that would mean `to` is an element or modifier, wheresas "next to" is the modifier.

**Pull request merged**